config/crds/migration_v1alpha1_migstorage.yaml

```diff
@@ -72,6 +72,8 @@ spec:
               type: object
             backupStorageProvider:
               type: string
+            refresh:
+              type: boolean
```

Comment:
@sseago do you recall if there's a reason why we need MigStorage to be refreshed? I thought this might have been put in after a conversation we had.

One benefit of updating the migstorage would be that migstorage validations will be re-run.

In reality though, I'm not sure if it's necessary to refresh the migstorage because migstorage has a background check for storage connectivity that should catch problems.

```
	// Watch for changes to cloud providers.
	err = c.Watch(
		&ProviderSource{
			Client:   mgr.GetClient(),
			Interval: time.Second * 30},
		&handler.EnqueueRequestForObject{})
	if err != nil {
		return err
	}
```

Reply:
@djwhatle 

> a MigStorage is responsible for local+remote BSLs and VSLs.

I may be missing something but if MigStorage creates remote BSLs/VSLs how does it get the cluster info where it needs to connect to? I need to dig on this one little more, will get back

Reply:
@alaypatel07 I was wrong, MigStorage does not handle this BSLs and VSLs. So I think the main argument in favor of updating the MigStorage would simply be to re-run the validations. I think we could remove this.